hydro_lang/src/stream.rs

```diff
@@ -64,16 +64,19 @@ impl MinOrder<TotalOrder> for NoOrder {
 /// - `L`: the location where the stream is being materialized
```

Comment:
These docs will need to be updated since we renamed the type parameters (same for other collection types).